rego/Meet.rego

```diff
@@ -38,7 +53,7 @@ if {
 tests contains {
     "PolicyId": "GWS.MEET.1.1v0.1",
     "Criticality": "Shall",
-    "ReportDetails": utils.ReportDetailsOUs(NonCompliantOUs1_1),
+    "ReportDetails": utils.ReportDetails(NonCompliantOUs1_1, []),
```

Comment:
Recommend adding a comment like this at least for the first setting, just so it's easier to understand what's going on.
```suggestion
    # Empty list in next line for non-compliant groups, as Meet settings can't be changed at the group level
    "ReportDetails": utils.ReportDetails(NonCompliantOUs1_1, []),
```